acceptance/helpers/catalog/misc.go

```diff
@@ -44,10 +44,10 @@ func NginxCatalogService(name string) models.CatalogService {
 		},
 		Values: values,
 		Settings: map[string]models.ChartSetting{
-			"ingress.enabled": models.ChartSetting{
+			"ingress.enabled": {
```

Comment:
Go allows the removal of the type name here ?

Reply:
Yes, it's inherited by the initial map instantiation (my linter-vscode was complaining about it)!

![image](https://github.com/epinio/epinio/assets/1763949/26e722ac-a806-47fd-a153-753c93e2c5c5)
![image](https://github.com/epinio/epinio/assets/1763949/eaebf5df-def0-4c71-abc2-26689046de58)